Tests/GitHubReferences.Tests.ps1

```diff
@@ -0,0 +1,173 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+.Synopsis
+   Tests for GitHubReferences.ps1 module
+#>
+
+[String] $root = Split-Path -Parent (Split-Path -Parent $MyInvocation.MyCommand.Path)
+. (Join-Path -Path $root -ChildPath 'Tests\Config\Settings.ps1')
+Import-Module -Name $root -Force
+
+function Initialize-AppVeyor
+{
+<#
+    .SYNOPSIS
+        Configures the tests to run with the authentication information stored in AppVeyor
+        (if that information exists in the environment).
+
+    .DESCRIPTION
+        Configures the tests to run with the authentication information stored in AppVeyor
+        (if that information exists in the environment).
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .NOTES
+        Internal-only helper method.
+
+        The only reason this exists is so that we can leverage CodeAnalysis.SuppressMessageAttribute,
+        which can only be applied to functions.
+
+        We call this immediately after the declaration so that AppVeyor initialization can heppen
+        (if applicable).
+
+#>
+    [CmdletBinding()]
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSAvoidUsingConvertToSecureStringWithPlainText", "", Justification="Needed to configure with the stored, encrypted string value in AppVeyor.")]
+    param()
+
+    if ($env:AppVeyor)
```

Comment:
I took this portion verbatim from other tests. I would be happy to move this Appveyor bit to a common util file which is then called from other tests. Thoughts?

Reply:
Others have mentioned that idea as well in previous PR's, but no-one ever got around to it.  I think it's a solid idea that I completely agree with, but it should be a different PR and not part of this one.  By all means, you're welcome to make that change as well, but please keep that as a separate PR.

Reply:
I did actually do this in a later change, so there will be some merging/resolving that you'll need to do as part of this PR.